### PR TITLE
[api][sims-#589] sql file name updated

### PR DESCRIPTION
**Pull request by ann-aot**
sql file name updated

### CODE REVIEW

File: sources/packages/api/src/database/migrations/1628287630225-OfferingIntensity.ts [hunk 1]
```diff
@@ -11,8 +11,6 @@ export class OfferingIntensity1628287630225 implements MigrationInterface {
   }
 
   public async down(queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query(
-      getSQLFileData("Remove-offering-intensity.sql", DIR),
-    );
+    await queryRunner.query(getSQLFileData("Drop-offering-intensity.sql", DIR));
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/database/migrations/1628287630225-OfferingIntensity.ts:14]: Is there a reason why one is using remove and the other is using drop?

Comment (ann-aot) [reply sources/packages/api/src/database/migrations/1628287630225-OfferingIntensity.ts:14]: remove is for a column, the drop is to drop the whole enum

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by abschwenker — APPROVED** [review]

**Pull request merged**